book/course/spack/existing.md

```diff
@@ -0,0 +1,244 @@
+# Using existing software
+
+Spack by default will quite happily just build everything itself, as long as a
+limited number of [system requirements](https://spack.readthedocs.io/en/latest/getting_started.html#system-prerequisites)
+are satisfied.  But you might want to use a compiler and MPI library from a
+host system.  This is often the case if packages take a long time to build, or
+if you've got access to optimised versions of software that you don't want to
+replicate yourself.
+
+In these circumstances, it's useful to be able to tell Spack about existing
+software, so it can use those instead of rebuilding them.  How much you tell
+Spack about your host system, and the installed software is up to you.  On one
+hand you can massively reduce how much software you need to build yourself, but
+on the other hand, you become much more dependent on the host system.  There's
+not one right answer.
+
+## Compilers
+
+Compilers are a bit different to other pieces of software.  Typically the
+easiest way to get Spack to use a system compiler is to make sure it's in your
+path, and then tell Spack to find it:
+
+```bash
```

Comment:
I'm confused by the below code snippet. You have this compiler already loaded as a module? Or is it just the default compiler in your PATH?

Maybe adding a `module load` call or `gcc -v`  before the `spack compiler add` would just help clarify that?

Reply:
You address this slightly below (line 31/32) but I think just for the snippets sake it might be worth adding just `gcc -v` or some text clarifying you've got gnu/8.5.0 installed as the default